### PR TITLE
Added devcontainers and updated requirements.txt

### PR DESCRIPTION
**Pull request by ghost**
Added the option to use devcontainers for easier creation of the dev environment. 

Also added the missing pytorch-lightning==1.9.3 version in the requirements.txt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
-Then run ```tensorboard --logdir logs --bind_all```
+# Installation
+1. Clone the repository
+```bash
+git clone https://github.com/RobotsGoMental/gtl_poc.git
+cd gtl_poc
+```
+2. Create a new conda environment with Python 3.9
+```bash
+conda create --name gtl python=3.9
```
Comment (vjekoslav) [inline README.md]: I wouldn't use "conda" but rather "venv" which is more vanilla, less opinionated. "conda" is very datasciency, so maybe it makes sense to move it completely, if those are our customers.

Personally I would rather use pipenv, poetry and venv before going for conda.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
\ No newline at end of file
+# Credits
+
+This project was created by Danko Nikolić, Davor Andrić, Vjekoslav Nikolić
+
+© 2023, Robots Go Mental, UG or its Affiliates. 
+
+# License
+
+This project is licensed under the [License Name].
```
Comment (vjekoslav) [inline README.md]: We have to figure out which license to use.

**Comment by vjekoslav** [comment]
Thanks for the PR Danijel!

File: requirements.txt [hunk 3]
```diff
@@ -1,113 +1,114 @@
...
+sniffio==1.3.0
+soupsieve==2.3.2.post1
+stack-data==0.6.2
+tensorboard==2.11.0
+tensorboard-data-server==0.6.1
+tensorboard-plugin-wit==1.8.1
+terminado==0.17.1
+tinycss2==1.2.1
+torch==1.13.1
+pytorch-lightning==1.9.3
```
Comment (vjekoslav) [inline requirements.txt:102]: You probably just wanted to add this missing dependency, but you also reformatted the whole file from Linux LF to Windows CRLF. Can you fix that?

File: README.md (hunk 1 above)
Comment (dandric) [reply README.md]: Let's go with venv as @vjekoslav proposed it's less opinionated.

File: README.md (hunk 2 above)
Comment (dandric) [reply README.md]: I would recommend to go with, https://creativecommons.org/licenses/by-nc-sa/4.0/

**Pull request merged**